### PR TITLE
StaticLLMPipeline: Adding python tests

### PR DESCRIPTION
**Pull request by TolyaTalamanov**

### CODE REVIEW

File: tests/python_tests/test_static_llm_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+    pipe = ov_genai.LLMPipeline(model_path, "NPU")
+    tokenizer = ov_genai.Tokenizer(model_path)
+    tokenized_input = tokenizer.encode(prompt)
+    # ignore_eos=True to ensure model will generate exactly num_tokens
+    encoded_results = pipe.generate(tokenized_input, max_new_tokens=num_tokens, ignore_eos=True)
+    assert len(encoded_results.tokens[0]) == num_tokens
+
+
+# FIXME: Known problem, output differs from stateful pipeline starting from 3rd prompt!
```
Comment (TolyaTalamanov) [inline tests/python_tests/test_static_llm_pipeline.py:134]: Failed ticket already

Comment (dmatveev) [reply tests/python_tests/test_static_llm_pipeline.py:134]: Filed

File: tests/python_tests/test_static_llm_pipeline.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        '1+1=',
+        'What is the previous answer?',
+        'Why is the Sun yellow?',
+        'What was my first question?'
+    ]
+
+    model_path = get_chat_models_lists()[0][1]
+
+    chat_history_stateful = generate_chat_history(model_path, "CPU", questions)
+    chat_history_static   = generate_chat_history(model_path, "NPU", questions)
```
Comment (dmatveev) [inline tests/python_tests/test_static_llm_pipeline.py:149]: so for this case, to only validate StaticLLMPipeline but not the whole stack down to NPUW/partitioning/CPU plugin (which sometimes goes crazy under partitioning), I'd recommend to pass a minimalistic NPUW config here: `NPUW_DEVICES: CPU, NPUW_ONLINE_PIPELINE:NONE`.

**Review by dmatveev — APPROVED** [review]
Let's go!

File: tests/python_tests/test_static_llm_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    get_chat_models_list,
+)
+
+
+# This test suite is designed specifically to validate the functionality and robustness of the StaticLLMPipeline on CPU.
+# On machines without NPU, the pipeline will automatically fall back to CPU, which aligns with the goal of these tests.
+# However, on machines where an NPU is available, the tests may fail due to NPU-specific compile or runtime issues,
+# which are outside the scope of this test suite and not its intended focus.
+# is_npu_available = 'NPU' in Core().get_available_devices()
+is_npu_available = False
```
Comment (Wovchena) [inline tests/python_tests/test_static_llm_pipeline.py:19]: Is it possible to create Core here and check if `NPU` is in available devices?

File: tests/python_tests/test_static_llm_pipeline.py (hunk 1 above)
Comment (Wovchena) [reply tests/python_tests/test_static_llm_pipeline.py:134]: Please, add a ticket number to the comment.

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: tests/python_tests/test_static_llm_pipeline.py (hunk 3 above)
Comment (TolyaTalamanov) [reply tests/python_tests/test_static_llm_pipeline.py:19]: Oh Jez, it's hardcode, sorry will fix

Comment (TolyaTalamanov) [reply tests/python_tests/test_static_llm_pipeline.py]: Removed this part completely, now fallback happens via config even on the platform with NPU

File: tests/python_tests/test_static_llm_pipeline.py (hunk 2 above)
Comment (TolyaTalamanov) [reply tests/python_tests/test_static_llm_pipeline.py]: Good point, done

File: tests/python_tests/test_static_llm_pipeline.py (hunk 1 above)
Comment (TolyaTalamanov) [reply tests/python_tests/test_static_llm_pipeline.py:128]: Done, I'd omit real JIRA name though

**Review by Wovchena — APPROVED** [review]

File: tests/python_tests/test_static_llm_pipeline.py (hunk 1 above)
Comment (Wovchena) [reply tests/python_tests/test_static_llm_pipeline.py:128]: Sure. You can only leave the number

**Comment by TolyaTalamanov** [comment]
Merged, but linux is disabled so far due segfault on CI (not reproducible locally)

**Pull request merged**